client/src/main/scala/drt/client/components/FlightsWithSplitsTable.scala

```diff
@@ -143,10 +143,11 @@ object FlightTableRow {
   implicit val stateReuse: Reusability[RowState] = Reusability.derive[RowState]
 
   def bestArrivalTime(f: Arrival): MillisSinceEpoch = {
+    def toSDateLikeOption(s: Option[MillisSinceEpoch]): Option[SDateLike] = s.flatMap(millis=>Try(SDate(millis)).toOption)
     val best = (
-      SDate.stringToSDateLikeOption(f.SchDT),
-      SDate.stringToSDateLikeOption(f.EstDT),
-      SDate.stringToSDateLikeOption(f.ActDT)
+      toSDateLikeOption(Some(f.Scheduled)),
```

Comment:
This function is probably no longer necessary seeing as you already have an SDateLike option here.